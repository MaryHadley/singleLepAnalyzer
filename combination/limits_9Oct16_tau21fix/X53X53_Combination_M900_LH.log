000 (1295376.000.000) 10/09 22:41:05 Job submitted from host: <192.168.63.249:41533>
...
001 (1295376.000.000) 10/09 22:41:26 Job executing on host: <192.168.63.240:42701>
...
006 (1295376.000.000) 10/09 22:41:34 Image size of job updated: 175144
	74  -  MemoryUsage of job (MB)
	75324  -  ResidentSetSize of job (KB)
...
006 (1295376.000.000) 10/09 22:46:34 Image size of job updated: 283552
	145  -  MemoryUsage of job (MB)
	147632  -  ResidentSetSize of job (KB)
...
006 (1295376.000.000) 10/10 00:01:42 Image size of job updated: 283552
	145  -  MemoryUsage of job (MB)
	147644  -  ResidentSetSize of job (KB)
...
006 (1295376.000.000) 10/10 18:18:37 Image size of job updated: 297832
	157  -  MemoryUsage of job (MB)
	160324  -  ResidentSetSize of job (KB)
...
005 (1295376.000.000) 10/10 18:31:10 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 19:47:36, Sys 0 00:00:22  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 19:47:36, Sys 0 00:00:22  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1592  -  Run Bytes Sent By Job
	306  -  Run Bytes Received By Job
	1592  -  Total Bytes Sent By Job
	306  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :    31842        1   3824814
	   Memory (MB)          :      157     3072      3072
...
