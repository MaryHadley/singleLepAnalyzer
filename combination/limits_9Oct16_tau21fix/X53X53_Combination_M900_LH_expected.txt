# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0     0.0956   0.048309    0.19941  0.065981    0.14045
